generators/client/templates/angular/src/main/webapp/app/shared/service/_pagination-util.service.ts

```diff
@@ -8,7 +8,10 @@ export class PaginationUtil  {
         if (sortArray.length > 1){
             return sort.split(',').slice(-1)[0] === 'asc';
         } else {
-            // default to true if no sort defined
+            sortArray = sort.split('%2C');
+            if (sortArray.length > 1){
+                return sort.split('%2C').slice(-1)[0] === 'asc';
+            }
```

Comment:
This code is essentially a duplicate of the rest of the code in this function. It should be refactored to be cleaner. Also, return `sortArray.slice(-1)[0] === 'asc';` unless there's a reason for it to split the string twice (I know that line was preexisting).